tests/tensor/test_nlinalg.py

```diff
@@ -280,6 +281,26 @@ def test_det_shape():
     assert tuple(det_shape.data) == ()
 
 
+def test_slogdet():
+    rng = np.random.default_rng(utt.fetch_seed())
+
+    r = rng.standard_normal((5, 5)).astype(config.floatX)
+    x = matrix()
+    f = pytensor.function([x], slogdet(x))
+    f_sign, f_det = f(r)
+    sign, det = np.linalg.slogdet(r)
+    assert np.equal(sign, f_sign)
+    assert np.allclose(det, f_det)
+
+
+def test_slogdet_shape():
```

Comment:
We don't really need a test for scalar output Ops, since scalar types can only have `()` shape

Reply:
Sure! Removed.